README.md

```diff
@@ -37,6 +37,39 @@ and treat as best.
 A Polkadot [collator](https://wiki.polkadot.network/docs/en/learn-collator) for the parachain is
 implemented by the `polkadot-parachain` binary (previously called `polkadot-collator`).
 
+### Relaychain Interaction
+To operate a parachain node a connection to the corresponding relaychain is necessary. This can be achieved in one of two ways:
+1. Run a full relaychain node within the parachain node (default)
+2. Connect to an external relaychain node via websocket RPC
+
+#### In-process Relaychain Node
+By default if an external relaychain node is not specified then a full relaychain node will be spawned within the same process.
+
+This node has all of the typical components of a normal Polkadot node, and will have to fully sync with the relaychain to work.
+
+##### Example command
+```shell=
+#                                                          In-process node with this chainspec is spawned
+#                                                                                |
+#                                                                    |-----------------------|
+polkadot-parachain --chain parachain-chainspec.json --tmp -- --chain relaychain-chainspec.json
+```
+
+#### External Relaychain Node
+You can connect to an external relaychain node via websocket RPC by using the  `--relay-chain-rpc-urls` command line argument. This option accepts one or more space-separated websocket URLs to a full relay chain node. By default only the first URL will be used, with the rest acting as a backup in case the connection to the first node will be lost.
```

Comment:
```suggestion
An external relaychain node can be connected via websocket RPC by using the  `--relay-chain-rpc-urls` command line argument. This option accepts one or more space-separated websocket URLs to a full relay chain node. By default only the first URL will be used, with the rest acting as a backup in case the connection to the first node will be lost.
```